### PR TITLE
fix: adds custom validator to url tile layers

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR includes:
- Adds custom validator for validating the custom layers url tiles text input.
- Fixes styles in custom layers form card.

### CODE REVIEW

**Comment by sorodrigo** [comment]
urltilevalidator

File: src/components/form/Form.js [hunk 1]
```diff
@@ -3,6 +3,20 @@ import Validation from 'react-validation';
 import ReactSelect from 'react-select';
 import { FormattedMessage } from 'react-intl';
 
+Object.assign(Validation.rules, {
```
Comment (edbrett) [inline src/components/form/Form.js]: This can be moved to validation-rules.js in constants.

**Review by edbrett — APPROVED** [review]
Coooooool 🌻

**Comment by j8seangel** [comment]
There are some url tiles that haven't .jpg or .png so I won't add this as required.

**Pull request merged**